app/controllers/casa_cases_controller.rb

```diff
@@ -124,6 +129,15 @@ def reactivate
     end
   end
 
+  def copy_court_orders
+    authorize @casa_case, :update_court_orders?
+    CasaCase.find_by_case_number(params[:case_number_cp]).case_court_orders.each do |court_order|
+      dup_court_order = court_order.dup
+      dup_court_order.save
```

Comment:
save can return true or false
this should probably be wrapped in a transaction

Reply:
absolutely you're right. I'll commit the changes :)